protocol/messenger_communities.go

```diff
@@ -1778,3 +1782,107 @@ func (m *Messenger) SyncCommunitySettings(ctx context.Context, settings *communi
 	chat.LastClockValue = clock
 	return m.saveChat(chat)
 }
+
+func (m *Messenger) ExtractDiscordDataFromImportFiles(filesToImport []string) (*discord.ExtractedData, map[string]*discord.ImportError) {
```

Comment:
@Samyoul Notice that I've changed the signature to return a `map[string]*discord.ImportError`

This is so that we know which `filePath` caused which errors (and can be quickly looked-up `O(1)` instead of iterating over a list of errors in the client.